pkg/kritis/metadata/containeranalysis/containeranalysis_int_test.go

```diff
@@ -155,15 +155,15 @@ func TestGetMultiplePages_Vulnerabilities(t *testing.T) {
 		t.Fatalf("Found err %s", err)
 	}
 
-	if len(vuln) <= 300 {
-		t.Errorf("Pagination error: only received results from first page.")
+	if len(vuln) <= 900 {
+		t.Errorf("Pagination error: did not receive results from the final page.")
```

Comment:
The error message can become incorrect when the vulnz count goes above 1000.

Reply:
Done.